tests/numerics/project_dgp_02.cc

```diff
@@ -17,9 +17,6 @@
 
 // check that VectorTools::project works for DGP elements correctly
 
-char logname[] = "output";
```

Comment:
i see now where that `char logname` comes from. I think this can indeed be removed, because we don't support comparing with files named other than `output`.